2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -135,8 +159,11 @@ public void resizeDefaultCapacity() {
         arrayList.add(33);
         arrayList.add(21);
 
-        assertThat(arrayList.get(6)).isEqualTo(21);
-        assertThat(arrayList.size()).isEqualTo(7);
+        int size = (int) getObjectByName("size");
```

Comment:
@erneman1 let's rename this test method to `addShoudResizeDefaultCapacityWhenArrayIsFull`